### ISSUE TITLE
Coronavirus FAQs: Vaccine Q updates week of 11.30.20

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#16719]
## Issue Description
We need to add a vaccine section and new questions to the page.

---
## Tasks
- [x] Edit vaccine FAQs (Danielle/Lauren)
- [x] Get FAQs page ready for changes (Danielle)
- [x] Receive approval on edited FAQs (Danielle/Lauren)
- [x] Finalize content with any received changes from approval (Danielle)
- [x] Copy review and publish (Randi)

## Acceptance Criteria
- [x] New vaccine section and questions published on live site

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by DanielleThierryUSDSVA** [comment]
Hi @bethpottsVADEPO and @RLHecht. I've added a new vaccine section with placeholder questions for the following:

When will there be an FDA-approved COVID-19 vaccine?
Will I be able to get a COVID-19 vaccine as soon as one is available?
Who is considered to be high risk for COVID-19?
Will I have to pay a copay for the COVID-19 vaccine?
Where can I get the latest accurate COVID-19 vaccine information? 

And I've removed the vaccine question in the top section since we'll now have  a dedicated section.

If the questions are approved before I get back, @lalexanderson-dsva will slack you and give you the copy to flow in.

Also, **the "more questions** is below the accordion in the top section and needs to be moved above it. my citrix cut out and i can't get in to make that update before I head out. Thank you!

Node id = 3014
